src/status_im/contexts/wallet/common/activity_tab/view.cljs

```diff
@@ -88,20 +90,62 @@
     ;; :mint            [mint-activity activity]
     nil))
 
+(defn- pressable-text
+  [{:keys [on-press text]}]
+  [rn/text
+   {:style    {:text-decoration-line :underline}
+    :on-press on-press} text])
+
 (defn view
   []
-  (let [theme         (quo.theme/use-theme)
-        activity-list (rf/sub [:wallet/activities-for-current-viewing-account])]
-    (if (empty? activity-list)
-      [empty-tab/view
-       {:title       (i18n/label :t/no-activity)
-        :description (i18n/label :t/empty-tab-description)
-        :image       (resources/get-themed-image :no-activity theme)}]
-      [rn/section-list
-       {:sections                       activity-list
-        :sticky-section-headers-enabled false
-        :style                          {:flex               1
-                                         :padding-horizontal 8}
-        :content-container-style        {:padding-bottom jump-to.constants/floating-shell-button-height}
-        :render-fn                      activity-item
-        :render-section-header-fn       (fn [{:keys [title]}] [quo/divider-date title])}])))
+  (let [theme                    (quo.theme/use-theme)
+        address                  (rf/sub [:wallet/current-viewing-account-address])
+        activity-list            (rf/sub [:wallet/activities-for-current-viewing-account])
+        open-eth-chain-explorer  (rn/use-callback
+                                  #(rf/dispatch [:wallet/navigate-to-chain-explorer
+                                                 {:address address
+                                                  :network constants/mainnet-network-name}])
+                                  [address])
+        open-oeth-chain-explorer (rn/use-callback
+                                  #(rf/dispatch [:wallet/navigate-to-chain-explorer
+                                                 {:address address
+                                                  :network constants/optimism-network-name}])
+                                  [address])
+        open-arb-chain-explorer  (rn/use-callback
+                                  #(rf/dispatch [:wallet/navigate-to-chain-explorer
+                                                 {:address address
+                                                  :network constants/arbitrum-network-name}])
+                                  [address])]
+    [:<>
```

Comment:
I'm not a big fan of wrapping a component in a fragment, if this component is mounted on a view with `:flex-direction :row` it'll bne shown completely different.

wdyt?